package.json

```diff
@@ -20,10 +20,12 @@
   "dependencies": {
     "@saucelabs/testcomposer": "^2.0.0",
     "@tsconfig/node20": "20.1.2",
+    "@types/shelljs": "^0.8.14",
```

Comment:
If it's listed under `devDependencies`, it won't be installed during the `npm ci --production` [step](https://github.com/saucelabs/sauce-testcafe-runner/blob/main/.github/workflows/test.yml#L82), leading to a failure in the `npm run build` step.
Although it's not a runtime dependency, a deeper solution would require adjustments to the pipeline.